types/xrm/index.d.ts

```diff
@@ -3023,6 +3039,12 @@ declare namespace Xrm {
              * @remarks Not available during the "on load" event of the form.
              */
             refresh(): void;
+            
```

Comment:
Somewhere here there is a whitespace formatting problem that makes your CI build to fail:
https://travis-ci.org/github/DefinitelyTyped/DefinitelyTyped/builds/684749850#L307

Reply:
Hi, yes I know.
I solved this issue and published the correction in the successive pull request named "Removed trailing spaces from previous modifications".

I don't know if it's the correct approach to solve a pull request issue with another pull request, if there's a better one any advice is appreciated. 

Thanks in advance.

Reply:
> I don't know if it's the correct approach to solve a pull request issue with another pull request, if there's a better one any advice is appreciated.

Nope, it won't work. One can make PR request against other code that already exists in some place in this project. At the moment both of your PR are abased on branches from your forks. Technically you could make a pull request on your own fork of this repo to update the other branch, etc. But that is an overkill.
More proper and accepted is to just update this original PR with correct changes added as new commit, pushed to update PR. The commits will be squashed later into a single merge commit, so no worries.
More:
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork
pt: 10 and 11

Reply:
Ok i think i got it: i closed the other PR and fixed my code pushing modification on this PR.
Now the Travis CI Build check is passed.

Sorry for the little mess i made and thank you for the support.